x/committee/legacy/v0_16/migrate.go

```diff
@@ -107,7 +108,7 @@ func migrateSubParamPermissions(permission v015committee.SubParamChangePermissio
 		}
 
 		// add new requirement for stability committee
-		if isStabilityCommittee {
+		if committeeID == 1 {
```

Comment:
thoughts on creating some vars in this file for these magic numbers so it's clear the targeting committees?

Reply:
Added in https://github.com/Kava-Labs/kava/pull/1132/commits/6f1b340758b6c52d4d5de874abe6d6812c141522